test-packages/type-tests/test/tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "strict": true,
     "noEmit": true
   },
-  "files": [],
```

Comment:
Does this work? Normally you always need either files, includes or excludes, I thought. Was this a problem?

Reply:
That's actually the problem why all the type tests were disabled. Because it's used like the includes, but nothing is included.

Reply:
Also, the example in the readme also has no
- includes
- excludes
- files